### PR TITLE
[price-service] Update docs + compose-file

### PR DESCRIPTION
**Pull request by ali-bahjati**
This PR updates price service README to explain more about the service (taken from gitbook) and adds some instructions on running it in testnet and mainnet (using docker compose). I found compose file the easiest because users/protocols do not need to understand the service fully.

Later we can point to this readme from gitbook as a guide to self-hosting the service.

### CODE REVIEW

File: third_party/pyth/price-service/README.md [hunk 1]
```diff
@@ -1,54 +1,58 @@
 # Pyth Price Service
 
-This service exposes a REST and WS api to provide latest attestation message for a price feed id.
+Pyth price service is a webservice that listens to the Wormhole Network for Pyth price updates and serves them via a
```
Comment (jayantk) [inline third_party/pyth/price-service/README.md:3]: The Pyth price service

File: third_party/pyth/price-service/README.md [hunk 2]
```diff
@@ -1,54 +1,58 @@
...
-```
-$ docker buildx build -f third_party/pyth/price-service/Dockerfile.price_service -t pyth_price_service .
-```
+## Wormhole Spy
 
-Run the spy_guardian docker container in TestNet:
+Price service depends on a Wormhole spy to stream Pyth messages from the Wormhole Network to it.
+[Spy](https://github.com/wormhole-foundation/wormhole/blob/main/node/cmd/spy/spy.go) is a component of a
+Wormhole guardian node that joins the Wormhole Network peer-to-peer network and listens to the Wormhole verified
+messages and streams the messages that are coming from certain emitters (e.g., Pyth data emitters) to its subscribers.
```
Comment (jayantk) [inline third_party/pyth/price-service/README.md:27]: I think you could convey the same information here in fewer words. Try to go through and simplify the sentences to their essence.

Comment (jayantk) [reply third_party/pyth/price-service/README.md:27]: Also "The price service" and "The Spy" at the beginning of these sentences.

Comment (jayantk) [reply third_party/pyth/price-service/README.md:27]: maybe add a second paragraph here explaining how the price service connects to the spy and how it's configured

File: third_party/pyth/price-service/README.md [hunk 3]
```diff
@@ -1,54 +1,58 @@
...
---bootstrap /dns4/wormhole-testnet-v2-bootstrap.certus.one/udp/8999/quic/p2p/12D3KooWBY9ty9CXLBXGQzMuqkziLntsVcyz4pk1zWaJRvJn6Mmt \
---network /wormhole/testnet/2/1
-```
+## Run
 
-Or run the spy_guardian docker container in MainNet:
-For the MainNet gossip network parameters, see https://github.com/wormhole-foundation/wormhole-networks/blob/master/mainnetv2/info.md
+To run Pyth price service, you need to have Wormhole spy running. You can simply run both price service
+and spy using provided mainnet and testnet docker compose files. To run the mainnet docker
+compose file run the following command:
```
Comment (jayantk) [inline third_party/pyth/price-service/README.md:33]: this paragraph is confusing because the first sentence makes it sound like the reader needs to run the spy themselves, but then explains that it's done for them. I'd rephrase to make the first sentence something like "The repository contains a docker-compose file to run both the price service and spy."

File: third_party/pyth/price-service/README.md [hunk 4]
```diff
@@ -1,54 +1,58 @@
...
---nodeKey /node.key --spyRPC "[::]:7073" \
---bootstrap <guardianNetworkBootstrapParameterForMainNet> \
---network <guardianNetworkPathForMainNet> \
+docker-compose up -f docker-compose.mainnet.yaml
 ```
 
-Then to run the pyth_price_service docker container using a config file called `${HOME}/pyth_price_service/env` and logging to directory `${HOME}/pyth_price_service/logs`, do the
-following:
+The compose files use a public release of Pyth price service and spy. If you wish to change the
+price service, you need to build an image for using it first.
```
Comment (jayantk) [inline third_party/pyth/price-service/README.md:40]: build an image and then presumably edit the compose file to point to it? what are the steps to update the compose file?

File: third_party/pyth/price-service/README.md [hunk 5]
```diff
@@ -1,54 +1,58 @@
...
+First, build the wasm files from the repo root like below. This command generates the wasm files necessary
+for parsing Pyth messages coming from Wormhole.
 
 ```
-$ docker run \
---volume=${HOME}/pyth_price_service:/var/pyth_price_service \
--e PYTH_PRICE_SERVICE_CONFIG=/var/pyth_price_service/env \
---network=host \
-pyth_price_service
+docker buildx build -f Dockerfile.wasm -o type=local,dest=. .
```
Comment (jayantk) [inline third_party/pyth/price-service/README.md:48]: this means from the root of `pyth-crosschain`? "repo root" is a little confusing since this readme is buried somewhere deep inside.

Comment (jayantk) [reply third_party/pyth/price-service/README.md:48]: Is there something the reader can check to validate that the WASM files were generated correctly? maybe mention where the files are written?

**Review by jayantk — COMMENTED** [review]
I left you some writing comments

File: third_party/pyth/price-service/README.md (hunk 5 above)
Comment (ali-bahjati) [reply third_party/pyth/price-service/README.md:48]: Yeah that is the repo root.

Comment (ali-bahjati) [reply third_party/pyth/price-service/README.md:48]: I'm not sure that be possible, it copies into some deep parts. I like to get rid of the wasm files. Wasm is typescript has a big overhead that I'm not sure is worth the benefits.

Comment (ali-bahjati) [reply third_party/pyth/price-service/README.md:48]: p.s: wormhole got rid of them too.

**Review by jayantk — APPROVED** [review]
